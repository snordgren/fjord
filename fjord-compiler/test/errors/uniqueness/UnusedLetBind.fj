module errors.UnusedLetBind

unusedX : Int
unusedX = 
  let x = 1 in 0
