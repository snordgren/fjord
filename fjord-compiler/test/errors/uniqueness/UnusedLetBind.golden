test/errors/uniqueness/UnusedLetBind.fj:5:3:
  |
5 |   let x = 1 in 0
  |   ^^^^^^^^^^^^^^^
too few usages of x
